evm/src/executor/abi.rs

```diff
@@ -50,6 +50,8 @@ ethers::contract::abigen!(
             expectRevert(bytes4)
             record()
             accesses(address)(bytes32[],bytes32[])
+            captureLogs()
```

Comment:
Thanks for working on this PR! Definitely interested in it but haven't looked closely yet. Only comment for now is: should we keep the nomenclature consistent with `vm.record`, and therefore call this `vm.recordLogs`? 

I think using "captured" for the other cheatcode makes sense since these are not slot accesses and therefore `access` terminology doesn't quite fit